### PR TITLE
Resolve 22 client name

### PR DESCRIPTION
**Pull request by ogdenstudios**
@baudoinalkali, honestly, my opinion is the best move here is to hardcode the slug itself. 

If you want to get more clever with it, you could use a package like [slugify](https://www.npmjs.com/package/slugify) to convert whitespace to hyphens, but you'd still have to handle the [CamelCase](https://en.wikipedia.org/wiki/Camel_case) names like `CarTalk`. 

This also works well because sometimes you may find your page slugs won't line up 1-to-1 with the company names, so setting it manually gives you a really good way to control it fine-grained. 

This is one of those instances where the value of programmatically generating a value may be juice that isn't really worth the squeeze. My guess is once you set these slugs once, they will change very rarely if often. It's probably worth the time to just write the slug yourself.

### CODE REVIEW

File: components/ClientContainer.tsx [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
+        <a href={`/clients/${props.slug}`}>
             <li className="portfolio1 rounded-md shadow-2xl m-auto w-64 ease-in-out bg-white hover:bg-alkaligrey-700 transition duration-500 transform hover:-translate-y-1 hover:scale-110" style={{ backgroundImage: `url("/images/${props.backgroundImg}")`, backgroundRepeat: "no-repeat", backgroundSize: "cover" }}>
                 <h3 className="font-open rounded-md text-3xl text-white pl-2 pt-80 pl-5 pb-4">{props.name}</h3>
             </li>
         </a>
     )
 }
-export default ClientContainers
\ No newline at end of file
+export default ClientContainer
```
Comment (ogdenstudios) [inline components/ClientContainer.tsx:10]: I renamed this for clarity. 

Technically, since Next uses [webpack under the hood](https://nextjs.org/docs/api-reference/next.config.js/custom-webpack-config), you can use the export syntax that looks like:

```js
export default ClientContainer
```

And in other files, you import it like: 

```js
import ClientContainer from '../components/ClientContainer.tsx`
```

But technically, this syntax isn't valid JavaScript. It's just syntactic sugar that Webpack provides. This doesn't matter for you because it works fine for your project and it's totally conventional. 

But the thing to keep in mind, is since you're using the `export default`, it *renames the exported item based on what you call it in the import item*. Which means you actually had this: 

```js
export default ClientContainers
```

```js
import ClientContainer from '../components/ClientContainer.tsx`
```

And even though you exported `ClientContainers` with the `s`, you imported it as `ClientContainer` without the `s`. It threw me for a loop because I wanted to see where it got used, and I tried searching the codebase in my editor - but I didn't find it because it was missing the `s`. 

Basically, the webpack default exports are *allowing you to change the name* of your components, which is confusing as an outside reader coming in and trying to follow the breadcrumbs. You should name your components in a consistent way in their file names, in their exported names, in the way you import them, and in the way you use them. That makes finding them much much easier, and reduces cognitive load of understanding the code paths.

File: yarn.lock [hunk 2]
```diff
@@ -496,6 +496,11 @@ classnames@2.2.6, classnames@^2.2.5:
   resolved "https://registry.yarnpkg.com/classnames/-/classnames-2.2.6.tgz#43935bffdd291f326dad0a205309b38d00f650ce"
   integrity sha512-JR/iSQOSt+LQIWwrwEzJ9uk0xfN3mTVYMwt1Ir5mUcSN6pU+V4zQFFaJsclJbPuAUQH+yfWef6tm7l1quW3C8Q==
 
+classnames@^2.2.6:
```
Comment (ogdenstudios) [inline yarn.lock:499]: Ignore these changes - I installed and uninstalled the slugify package and it edited this file. It's perfectly fine, but it doesn't make a difference with regard to the issue this is addressing.

File: components/ClientContainer.tsx (hunk 1 above)
Comment (baudoinalkali) [reply components/ClientContainer.tsx:10]: @ogdenstudios

I'll just go with the slug option.
